app/views/content/shared/subject-pages/_adviser-promo.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.with_left_section(classes: %w[tta-background]) %>
+  <% promo.with_right_section(heading: "Get free one-to-one support") do %>
+    <p>An adviser with years of teaching experience can help you to become <%= @front_matter["subject"].indefinite_article %> <%= @front_matter["subject"] %> teacher. Chat by phone, text, or email as little or often as you need.</p>
```

Comment:
Check the docs of the indefinite article gem, you can simplify this even further with `.with_indefinite_article` to get the article and the subject in one go

Reply:
Thank you! Works a treat!